run-system-tests/action.yml

```diff
@@ -197,6 +197,33 @@ runs:
         (./zwallet version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZWallet Version Info:/") || true
         (./zbox version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZBox Version Info:/") || true
 
+    - name: "Stake blobbers using above built binaries"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        echo '#!/bin/bash
+        echo
+        echo "==========================================================="
+        echo "STAKING BLOBBERS USING ZWALLETCLI & ZBOXCLI BINARIES"
```

Comment:
I would add this to the deployment action rather than system test - it really is a deployment responsibility and not a responsibility of the test run
That also gives you the added benefit of being able to re-attempt the deployment (see my other comment) if this fails for some reason

Reply:
This is true and I tried the same initially, but, this needs the build binaries job completed before it. We can move that too to deploy-0chain, but that is not called for runs against existing networks. One way to go about this is: build binaries during deploy-0chain, rm binaries in system-tests (if any) and build again. Should I go with that?

Reply:
Done